app/javascript/packs/application.js

```diff
@@ -7,11 +7,13 @@
 // To reference this file, add <%= javascript_pack_tag 'application' %> to the appropriate
 // layout file, like app/views/layouts/application.html.erb
 
-import '../src/turbolinks';
+// Yarn globals that don't require much beyond an import should live in here.
+import '../src/core_dependencies';
```

Comment:
It's started to make less sense to me to put global imports in separate files after the third, so I've moved them to a `core_dependencies` file responsible for, turbolinks/bootstrap/fontawesome/any other things that are just `import 'library'`.